google_assistant/rootfs/etc/services.d/google-assistant/run

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/with-contenv bashio
+# shellcheck shell=bash
```

Comment:
Let's not touch google_assistant to avoid CI run. We'll have to deprecate this add-on.

```suggestion
```